src/prompt_siren/attacks/agentdojo_attack.py

```diff
@@ -116,7 +117,11 @@ async def attack(
                         user=self._user_name,
                         model=agent.get_agent_name(),
```

Comment:
We should probably make these extra fields part of the attack config? Different attacks might want to define their own custom templated fields.

Reply:
Perhaps we can have as part of the config a list of fields? Or maybe we should use a jinja template for this?